### ISSUE TITLE
OverflowError: cannot fit 'int' into an index-sized integer

### ISSUE THREAD

**Issue by oyrq** [ChenYingpeng/caffe-yolov3#25]
Hi,I have this problem : 

Traceback (most recent call last):
  File "yolov3_darknet2caffe.py", line 533, in <module>
    darknet2caffe(cfgfile, weightfile, protofile, caffemodel)
  File "yolov3_darknet2caffe.py", line 19, in darknet2caffe
    params = net.params
  File "/home/oyrq/caffe/python/caffe/pycaffe.py", line 67, in _Net_params
    self._layer_names, self.layers)
  File "/home/oyrq/caffe/python/caffe/pycaffe.py", line 68, in <listcomp>
    if len(lr.blobs) > 0])
OverflowError: cannot fit 'int' into an index-sized integer
Segmentation fault (core dumped)

Thanks!

**Comment by JubilantJerry** [comment]
I got this, and this solved it for me at least:

https://github.com/BVLC/caffe/issues/2437#issuecomment-184984016

TL;DR rebuild your caffe
